hi.js

```diff
@@ -0,0 +1,688 @@
+module.exports = `
+menu.btn.ranked रैंक का खेल
+menu.btn.shop दुकान
+menu.btn.social हब
+menu.btn.maps खेल
+menu.btn.mods मॉड
+menu.btn.settings सेटिंग
+menu.btn.customize रूप बदले
+menu.btn.invite आमंत्रण
+menu.btn.invite.clicked कॉपी
+menu.btn.join ज्वाइन
+menu.btn.host होस्ट खेल
+menu.btn.browser सर्वर
+generic.wait कृपया प्रतीक्षा करें...
+generic.loading लोड हो रहा है । ..
+generic.none कुछ भी नहीं
+generic.connecting कनेक्टिंग...
+generic.select चयन करें
+generic.delete हटाएं
+generic.save रूप बदले
+generic.load लोड
+generic.search खोजिए
+generic.leave छोड़ो
+generic.default डिफ़ॉल्ट
+generic.submit सबमिट करें
+generic.create बनाएँ
+generic.random रैंडम
+generic.failed असफल
+generic.success सफलता
+generic.name नाम
+generic.level स्तर
+generic.score स्कोर
+generic.off बंद
+generic.defeat हार
+generic.victory विजय
+generic.matchover मैच ओवर
+generic.matchaband मैच छोड़ दिया
+generic.follow फॉलो
+generic.unfollow अनफॉलो
+generic.followers अनुयायी
+generic.following निम्नलिखित
+generic.games खेल
+generic.wins जीत
+generic.lose घाटा
+generic.clan क्लैन
+generic.games.played खेले गये खेल
+generic.games.won खेल जीते
+generic.games.lost खेल हारे
+generic.timep कितना समय खेला
+generic.kills हत्या
+generic.deaths मौतें
+generic.kdr KDR
+generic.time समय
+generic.logout लॉगआउट
+generic.logout.sessions सभी सत्रों का लॉगआउट
+generic.username उपयोगकर्ता नाम
+generic.password पासवर्ड
+generic.register रजिस्टर
+generic.login लॉगिन
+generic.accuracy सटीकता
+player.waiting खिलाड़ियों का इंतजार
+player.died तुम मर गए
+player.killed द्वारा मारे गए
+player.killstreak {0} एक {1} मार लकीर पर
+player.inactive इनैक्टिव
+player.disconnect डिस्कनेक्ट
+timer.end अगले दौर में {0}
+leaderboard.empty खाली लॉबी
+popup.checkpoint चैकपौइंट
+app.play खेलने के लिए क्लिक करें
+app.spectate स्पैक्टेट करे
+app.play.mobile खेलने के लिए टैप करें
+app.spectate.mobile स्पैक्टेट करे
+app.play.controller प्रेस शुरू
+app.spectate.controller स्पैक्टेट करे
+purchase.error ऐरर
+purchase.error.click यहां क्लिक करें  </a> और फिर से प्रयास करें!
+purchase.error.limit आप अपनी दैनिक लेनदेन सीमा को पार कर चुकें हैं
+settings.reset रीसेट
+settings.reset.confirm क्या आप वाकई अपनी सभी सेटिंग्स रीसेट करना चाहते हैं?
+settings.import इम्पोर्ट
+settings.export ऐक्सपोर्ट
+settings.none कोई सेटिंग नहीं मिली
+settings.backup बैकअप सेटिंग्स
+settings.controls.change Keybinds
+settings.controls.press किसी भी कुंजी को दबाएं
+settings.local.header Localization <span id="requiresRestart"><span style="color: #eb5656">*</span> requires restart</span>
```

Comment:
Is this not translatable ? Or best to be kept in English?

Reply:
I would suggest keeping this in English only, cuz there is no intelligible alternate in Hindi.

Reply:
I tried finding alternatives but no luck, so I decided to keep it in English.